Mirror/Weaver/UNetBehaviourProcessor.cs

```diff
@@ -2036,7 +2036,8 @@ void ProcessSyncVars()
                             fieldModuleName != Weaver.m_UnityAssemblyDefinition.MainModule.Name &&
                             fieldModuleName != Weaver.m_UNetAssemblyDefinition.MainModule.Name &&
                             fieldModuleName != Weaver.corLib.Name &&
-                            fieldModuleName != "System.Runtime.dll" // this is only for Metro, built-in types are not in corlib on metro
+                            fieldModuleName != "System.Runtime.dll" && // this is only for Metro, built-in types are not in corlib on metro
+                            fieldModuleName != "netstandard.dll" // handle built-in types when weaving new C#7 compiler assemblies
```

Comment:
Can you give an example of a type in netstandard.dll that is needed? I haven't tried that Unity version yet

Reply:
In my case:

```csharp
[SyncVar]
public string DisplayName;

[SyncVar]
public bool IsAlive;
```

Results in `System.String` and `System.Boolean` being referenced from `netstandard.dll`

Reply:
I'm not an expert on the topic, but you can refer to the comment here for an explanation why this occurs
https://github.com/dotnet/standard/issues/146#issuecomment-271105398

Reply:
Thanks!